src/localplayer.cpp

```diff
@@ -91,6 +91,21 @@ LocalPlayer::~LocalPlayer()
 {
 }
 
+static aabb3f getTopBoundingBox(const std::vector<aabb3f> &nodeboxes, float max_d=1/16*BS)
```

Comment:
There's no mention of leaving them out in https://www.kernel.org/doc/Documentation/process/coding-style.rst spaces also make things much more readable.